index.html

```diff
@@ -808,6 +811,90 @@ <h3>
           </p>
         </section>
       </section>
+      <section>
+        <h3>
+          `id` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest" id="identity">id</dfn></code> member is a <a>string</a> that
```

Comment:
I wonder if we should rename this "app_id" instead? I think "id" is overly generic and app_id makes it more clear that this is for the whole "app", not the id of the manifest.

Reply:
In the case of miniapps, all the vendors agreed to use `app_id` as [identifier for the app](https://w3c.github.io/miniapp-manifest/#dfn-app_id).

Reply:
Oh, nice! Looks like we can share the name of this member.

Reply:
I was thinking about it too. But I feel other fields are for the whole "app" too? eg: scope / start_url?

Reply:
I'm not very familiar with the MiniApps work. Are we intentionally aligning with them / is there some assumed compatibility between the two standards? Do we try to align opportunistically (i.e. if there's something that "feels the same" we try to align but we don't go out of our way to make it compatible)?

I think we could align with them on the name, but also we have a somewhat different semantics for our ID.

Ignoring MiniApps, I would have a slight preference for `id` over `app_id`, echoing what @philloooo said. It's called the "Web App Manifest", so "app" is implied, and we could similarly argue that it should be called `app_scope`, `app_name`, etc. So my vote is for `id`.

Reply:
> I'm not very familiar with the MiniApps work. Are we intentionally aligning with them / is there some assumed compatibility between the two standards? Do we try to align opportunistically (i.e. if there's something that "feels the same" we try to align but we don't go out of our way to make it compatible)?

The other way round :) MiniApp is trying to reuse/align with the existing standards. As a first approach, the MiniApp vendors agreed to extend the Web App Manifest. 

The purpose and semantics of the MiniApp's [`app_id`](https://w3c.github.io/miniapp-manifest/#dfn-app_id) member are the same (identify the app univocally), but most of the MiniApp vendors don't use URLs to identify the app. I can bring this discussion to the MiniApp WG and see if this member could be reused.

I agree with you @mgiuca @philloooo that the term `id` would be enough since `app` may be redundant.

Reply:
Alright, let's stick with `id`.